### PR TITLE
feat(general): fixed Makefile paths

### PR DESCRIPTION
**Pull request by WikiRik**
The Makefile paths were not updated yet. @serge1peshcoff I am not sure if I changed the backup_core correctly.
I also changed localhost into 127.0.0.1 in Vagrantfile, since that caused an error for me (both Windows and Linux)

### CODE REVIEW

**Comment by WikiRik** [comment]
Btw, couldn't push directly to this repo, so I used a fork. Not sure if tests will run this way

File: Makefile
Comment (serge1peshcoff) [inline Makefile]: in `postgres-core-`, there's an extra `-`

Comment (WikiRik) [reply Makefile]: Oops, fixed

**Review by linuxbandit — APPROVED** [review]
For next times, don't forget to separate in different commits unrelated things (in this case, vagrant) ;)

**Comment by linuxbandit** [comment]
@serge1peshcoff  what do you think this check anomaly is? I mean, going to the tab "checks" shows empty, I expect something to be there for consistency with this page (even though we have configured tests to not run for forks - and even there, can you see options for running most commands except the ones with passwords?)

**Comment by serge1peshcoff** [comment]
okay, this is most likely due to CircleCI not running PR checks for forked PRs:
image

**Comment by serge1peshcoff** [comment]
I've made myself an admin of AEGEE/MyAEGEE and disabled some administrators restrictions in order to be able to merge this, will disable it after merging.
@WikiRik I've also given you the write access, so it's better to push to this repo directly.

**Comment by linuxbandit** [comment]
but this is silly, CircleCI does not tell github to not wait?

**Comment by serge1peshcoff** [comment]
CircleCI doesn't send the checks statuses because they are not executed,
and they are required, so is impossible to merge stuff this way.

**Comment by WikiRik** [comment]
> I've made myself an admin of AEGEE/MyAEGEE and disabled some administrators restrictions in order to be able to merge this, will disable it after merging.
> @WikiRik I've also given you the write access, so it's better to push to this repo directly.

Awesome, thanks!

**Pull request merged**
